static/src/stylesheets/module/content-garnett/_types.scss

```diff
@@ -823,11 +823,25 @@ $quote-mark: 35px;
 .content--type-comment {
     .content__main {
         background-color: $opinion-garnett-background;
+        height: 100%;
```

Comment:
Is this really necessary? 😮

Reply:
It was the cleanest fix I could come up with, there's this weird empty div with a margin top that's added to every article to create spacing at the bottom, I suppose we could change the margin-top to a padding or height value? Not sure if that's better though

<img width="1070" alt="screen shot 2018-01-19 at 12 41 57" src="https://user-images.githubusercontent.com/8453924/35151434-32954406-fd16-11e7-98c2-c4383788d7c7.png">